testsuite/generate-format/tests/staged/network.yaml

```diff
@@ -103,6 +103,8 @@ Protocol:
     8:
       Handshake:
         NEWTYPE: U8
+    9:
+      PeerIdExchange: UNIT
```

Comment:
what's a UNIT oO?

Reply:
yeah `"/ln-peerid-ex` has no arguments, hence "()" is the enum value I guess

I makes sense that these two examples are isomorphic

```rust
enum Foo {
    Bar,
}

enum Foo {
    Bar(()),
}
```